plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/quickfix/N4JSQuickfixProvider.xtend

```diff
@@ -778,6 +779,8 @@ class N4JSQuickfixProvider extends AbstractN4JSQuickfixProvider {
 				new ProgressMonitorDialog(UIUtils.shell).run(true, true, [monitor |
 					try {
 						libraryManager.synchronizeNpms(monitor);
+					} catch (OperationCanceledException e) {
```

Comment:
ignore ? not even log the exception?

Reply:
also, why not multi-catch?

Reply:
a) at other places, we do nothing either, so I thought that okay. Anyway, I aligned the code more to other places, i.g. `MaintenanceActionsButtonListener:114`
b) xtend does not support multi-catch